source/content/overview/io.md

```diff
@@ -0,0 +1,625 @@
+     you can also insert your own.
+ -   Each workbook can only describe data for one `Database`. Multiple
+     `Database` sections will raise an error!
+ -   Input exchanges must be linked using the data provided in the
+     spreadsheet. That means that you may need to provide an activity
+     name and a reference product when trying to link against
+     ecoinvent.
+ -   Linking is done using the column headings exactly as provided. You
+     should check to make sure the capitalization, spelling, and
+     spacing are consistent with the attributes in the background
+     databases you want to link against.
+ -   Worksheet names don\'t have any special meaning to the importer,
+     and are only used to indicate where errors occurred.
+
+Parameters are a bit different - as they are trickier to handle, their
+sections are stricter:
+
+ -   All parameter sections must have the column \"name\". Names should
+     be unique within a parameter section.
+ -   Stochastic variables should have the usual uncertainty columns,
+     e.g. uncertainty type, loc, scale, minimum, maximum.
+ -   Variables which are defined by formulas should have the column
+ -   Parameters can have an \"amount\" column, but if there is also a
+     \"formula\" column, the amount will be overwritten on import as
+     the formula is evaluated.
+ -   Formulas should be for consumption by Python, not Excel; they will
+     be interpreted by `asteval`(https://newville.github.io/asteval/).
+ -   The `Database`, `Project parameters`, and `Database parameters`
+     don\'t have to be in any order or in any particular worksheet.
+     There can be multiple `Project parameters` and
+     `Database parameters` sections, they will be concatenated.
+
+The following data transformations are applied by the `ExcelImporter`:
+
+ -   Numbers are translated from text into actual numbers.
+ -   Tuples, separated in the cell by the `::` string, are
+     reconstructed, i.e. \"this::example\" would become
+     `("this", "example")`.
+ -   The strings \"True\" and \"False\" (by themselves, not as part of
+     a larger string) are transformed to their respective boolean
+     values.
+ -   Fields with the value `(Unknown)` are omitted on a per-row basis.
+
+### Importing LCIA methods from the standard Excel template
+
+Proper implementation of life cycle impact assessment methods requires a complete set of metadata. However, for simple LCIA methods, Brightway
+has a simplified importer for Excel and CSV files: `bw2io.ExcelLCIAImporter` and `bw2io.CSVLCIAImporter`. These function the same as other importer (i.e. you extract data, apply strategies until you are satisfied with the matching, and write the modified data). There are example [Excel workbooks](https://github.com/brightway-lca/brightway2-io/raw/legacy/tests/fixtures/excel/lcia.xlsx) and [CSV files](https://github.com/brightway-lca/brightway2-io/raw/legacy/tests/fixtures/csv/lcia.csv).
+
+Please note the following:
```

Comment:
Should go into admonition box.

Reply:
This was all copied over and will be rewritten.